docs/reference/cli-config.rst

```diff
@@ -0,0 +1,39 @@
+Command Line Interface Configuration
+====================================
+
+The Testflinger CLI can be configured on the command line using the parameters described in ``testflinger --help``. However, sometimes it also supports using environment variables, or reading values from a configuration file. This can be helpful for CI/CD environments, or for setting up a config with different default values read from a config file.
+
+The configuration file is read from ``$XDG_CONFIG_HOME/testflinger-cli.conf`` by default, but this can be overridden with the ``--configfile`` parameter.
+
+You can show the current values stored in the config file by running ``testflinger config``. If no value is found on the command line, config file, or environment variable, then a safe default value will be used.
+
+To set a configuration value in the config file, you can edit it directly or use ``testflinger config key=value``.
```

Comment:
Does this command update only the default config file?

Reply:
I asked because I suppose `testflinger config` only outputs the configs in the default file. If  `--configfile` can be used, then perhaps I can check the current valid configs in the temporary file, and update it (it's a creative usage, but just wanted to see if it's technically valid)